commit 42548bd7137af5cb7aa9664b4a65a3b469aee366
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2019-03-13T12:55:43+00:00 (7 months ago)
    
    MANTA-4163 improved PG autovacuum msg
    Reviewed by: Richard Bradley <richard.bradley@joyent.com>
    Approved by: Richard Bradley <richard.bradley@joyent.com>

diff --git a/deps/postgresql96 b/deps/postgresql96
index fda6203..96cfa73 160000
--- a/deps/postgresql96
+++ b/deps/postgresql96
@@ -1 +1 @@
-Subproject commit fda6203fca5c8a4bd3dda2dc725cf02134e8725e
+Subproject commit 96cfa7397d8dc0a9db3f8083ef4ad399be88fc0f
